spec/services/store_sp_metadata_in_session_spec.rb

```diff
@@ -32,232 +32,19 @@
         end
       end
 
-      before do
```

Comment:
We have dramatically reduced the role of `StoreSpMetadataInSession` by moving to the `AuthnContextResolver`. I have reduced the complexity of the specs in response to this.